### PR TITLE
refactor(InputGroup): refactor css

### LINKED ISSUE #993: Use a configuration first method (modifiers and states) for InputGroup
Relates directly back to #273

### PR DESCRIPTION
**Pull request by christiemolloy**
Closes #993 . Refactors CSS

### CODE REVIEW

File: src/patternfly/components/InputGroup/input-group.scss [hunk 1]
```diff
@@ -3,17 +3,18 @@
...
-  --pf-c-input-group__text--BorderRightColor: var(--pf-global--BorderColor--light-200);
-  --pf-c-input-group__text--BorderLeftColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-input-group__text--BorderXColor: var(--pf-global--BorderColor--light-200);
   --pf-c-input-group__text--BorderBottomColor: var(--pf-global--BorderColor--dark);
   --pf-c-input-group__text--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
   --pf-c-input-group__textarea--MinHeight: var(--pf-global--spacer--xl);
+
+  // Form Control
   --pf-c-input-group--form-control-invalid--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-input-group--form-control--MarginRight: 1px;
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/input-group.scss]: can you rename these vars to have `c-form-control`?

File: src/patternfly/components/InputGroup/input-group.scss [hunk 2]
```diff
@@ -3,17 +3,18 @@
...
   --pf-c-input-group__text--BorderTopColor: var(--pf-global--BorderColor--light-200);
-  --pf-c-input-group__text--BorderRightColor: var(--pf-global--BorderColor--light-200);
-  --pf-c-input-group__text--BorderLeftColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-input-group__text--BorderXColor: var(--pf-global--BorderColor--light-200);
   --pf-c-input-group__text--BorderBottomColor: var(--pf-global--BorderColor--dark);
   --pf-c-input-group__text--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
   --pf-c-input-group__textarea--MinHeight: var(--pf-global--spacer--xl);
+
+  // Form Control
   --pf-c-input-group--form-control-invalid--ZIndex: var(--pf-global--ZIndex--xs);
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/input-group.scss]: can you change this to `c-form-control--invalid`?

File: src/patternfly/components/InputGroup/input-group.scss [hunk 3]
```diff
@@ -82,13 +82,13 @@
...
+    padding-right: var(--pf-c-input-group__text--PaddingX);
+    padding-left: var(--pf-c-input-group__text--PaddingX);
     font-size: var(--pf-c-input-group__text--FontSize);
     color: var(--pf-c-input-group__text--Color);
     text-align: center;
     background-color: var(--pf-c-input-group__text--BackgroundColor);
-    border: var(--pf-c-input-group__text--BorderSize) solid;
-    border-color: var(--pf-c-input-group__text--BorderTopColor) var(--pf-c-input-group__text--BorderRightColor) var(--pf-c-input-group__text--BorderBottomColor) var(--pf-c-input-group__text--BorderLeftColor);
+    border: var(--pf-c-input-group__text--BorderWidth) solid;
+    border-color: var(--pf-c-input-group__text--BorderTopColor) var(--pf-c-input-group__text--BorderXColor) var(--pf-c-input-group__text--BorderBottomColor) var(--pf-c-input-group__text--BorderXColor);
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/input-group.scss]:
```suggestion
    border-color: var(--pf-c-input-group__text--BorderTopColor) var(--pf-c-input-group__text--BorderXColor) var(--pf-c-input-group__text--BorderBottomColor);
```

**Comment by mcoker** [comment]
looks great! just a few suggestions.

**Comment by christiemolloy** [comment]
Thanks for the review. Made changes :)

**Pull request merged**